service/src/test/java/bio/terra/workspace/common/fixtures/ControlledGcpResourceFixtures.java

```diff
@@ -347,4 +350,57 @@ public static ControlledAiNotebookInstanceResource.Builder makeDefaultAiNotebook
         .location("us-east1-b")
         .projectId("my-project-id");
   }
+
+  // GCE instance
+
+  public static final ApiGcpGceUpdateParameters GCE_INSTANCE_PREV_PARAMETERS =
+      new ApiGcpGceUpdateParameters()
+          .metadata(ImmutableMap.of("sky", "blue", "rose", "red", "foo", "bar2", "count", "0"));
+
+  public static final ApiGcpGceUpdateParameters GCE_INSTANCE_UPDATE_PARAMETERS =
+      new ApiGcpGceUpdateParameters().metadata(ImmutableMap.of("foo", "bar", "count", "3"));
+
+  public static ApiGcpGceInstanceCreationParameters defaultGceInstanceCreationParameters() {
+    return new ApiGcpGceInstanceCreationParameters()
+        .instanceId(TestUtils.appendRandomNumber("default-instance-id"))
+        .zone("us-central1-a")
+        .machineType("zones/us-central1-a/machineTypes/n1-standard-1")
+        .vmImage("projects/debian-cloud/global/images/family/debian-11");
+  }
+
+  /**
+   * Returns a {@link ControlledGceInstanceResource.Builder} that is ready to be built.
+   *
+   * <p>Tests should not rely on any particular value for the fields returned by this function and
+   * instead override the values that they care about.
+   */
+  public static ControlledResourceFields.Builder makeGceInstanceCommonFieldsBuilder() {
+    return ControlledResourceFields.builder()
+        .workspaceUuid(UUID.randomUUID())
+        .resourceId(UUID.randomUUID())
+        .name(TestUtils.appendRandomNumber("my-instance"))
+        .description("my description")
+        .cloningInstructions(CloningInstructions.COPY_NOTHING)
+        .assignedUser("myusername@mydomain.mine")
+        .accessScope(AccessScopeType.ACCESS_SCOPE_PRIVATE)
+        .managedBy(ManagedByType.MANAGED_BY_USER)
+        .createdByEmail(MockMvcUtils.DEFAULT_USER_EMAIL)
+        .region(DEFAULT_RESOURCE_REGION);
+  }
+
+  public static ControlledGceInstanceResource.Builder makeDefaultGceInstance() {
+    return ControlledGceInstanceResource.builder()
+        .common(makeGceInstanceCommonFieldsBuilder().build())
+        .instanceId(TestUtils.appendRandomNumber("my-cloud-id"))
+        .zone("us-east1-b")
```

Comment:
is DEFAULT_RESOURCE_REGION at line 388 us-east1?

Reply:
it's us-central1, but I don't think it's used for anything here